bgpd/bgp_vty.c

```diff
@@ -6433,11 +6477,21 @@ DEFUN (clear_bgp_instance_ipv6_safi_prefix,
        "IPv6 prefix\n")
 {
 	int idx_word = 3;
-	int idx_safi = 5;
-	int idx_ipv6_prefixlen = 7;
+	int idx_safi;
+	int idx_ipv6_prefix = 7;
```

Comment:
since you can avoid using indices entirely here, I would do so, as it'll avoid introducing bugs in the future when someone modifies the command and forgets to update the indices...you're already using `argv_find()` so why not just get rid of the indices entirely?

Reply:
ok